sql/src/test/java/io/crate/protocols/postgres/QueryStringSplitterTest.java

```diff
@@ -32,6 +32,7 @@
     @Test
     public void testEmptyQuery() {
         assertThat(QueryStringSplitter.splitQuery(";"), contains(";"));
+        assertThat(QueryStringSplitter.splitQuery(";; ;"), contains(";", ";", " ;"));
```

Comment:
Is this correct? meaning that if we have ```select 1;;;;select2;;;``` shouldn't this be split only in 2 queries: `select 1` and `select 2` ?

Reply:
Since queries are delimited by `;`, I believe this is the correct way to handle this. The goal of the parser is not to rewrite the statements, but to split them.

Reply:
Maybe it's a minor detail. My question is only to be fully compatible with what postgres does.
If `select 1;;;select 2` executes successfully in postgres but not in CrateDB maybe we need to also handle this.